undertow/src/main/java/org/wildfly/extension/undertow/Constants.java

```diff
@@ -46,6 +46,7 @@
     String HTTP_INVOKER = "http-invoker";
     String LISTENER = "listener";
     String INSTANCE_ID = "instance-id";
+    String UUID_INSTANCE_ID = "uuid-instance-id";
```

Comment:
IMO, this attribute name is no longer sufficiently descriptive, since, when enabled, the generated instance-id has nothing to do with a UUID.
Perhaps "obfuscate-instance-id"?